com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/errors.py

```diff
@@ -0,0 +1,2 @@
+class ViewNotFoundError(Exception):
+    """Raised when the SWS view element cannot be located."""
```

Comment:
>As a user of the api I just care about a view
As a user of the topology API, this is correct. But if this is moved to streamsx.utility, in context, I think "the SWS view element cannot be located" is appropriate since that's the layer at which we're interfacing with SWS REST elements.